src/components/ui/stepper.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { cn } from '@/lib/utils';
+
+interface IStep {
+  currentStep: number;
+  maxSteps: number;
+}
+
+const Stepper: React.FC<IStep> = ({ currentStep, maxSteps }: IStep) => {
+  const steps: number[] = Array.from(
+    { length: maxSteps },
+    (_, index) => index + 1,
+  );
+
+  const width = `${(100 / (maxSteps - 1)) * (currentStep - 1)}%`;
+
+  return (
+    <div className="w-full max-w-2xl mx-auto px-1 pb-10">
+      <div className="flex justify-between relative before:bg-slate-300 before:absolute before:h-[0.125rem] before:top-1/2 before:transform-y-1/2 before:w-full before:left-0">
```

Comment:
The font-weight is 500 so could you add `font-medium`?
<img width="623" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/144519746/4db374f3-2273-479f-a2c2-9d40019b25fa">